internal/pkg/levee/propagation/propagation.go

```diff
@@ -43,22 +43,22 @@ type Propagation struct {
 
 // Dfs performs a depth-first search of the graph formed by SSA Referrers and
 // Operands relationships, beginning at the given root node.
-func Dfs(n ssa.Node, conf *config.Config, taggedFields fieldtags.ResultType) Propagation {
+func PropagateTaint(n ssa.Node, conf *config.Config, taggedFields fieldtags.ResultType) Propagation {
 	record := Propagation{
```

Comment:
This isn't directly related to your changes, but since you're renaming things, consider renaming this variable to `prop` instead of `record`.

Reply:
Done.